src/pb_stub.cc

```diff
@@ -1611,7 +1611,17 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
       .export_values();
 
   py::class_<InferenceTrace, std::shared_ptr<InferenceTrace>>(
-      module, "InferenceTrace");
+      module, "InferenceTrace")
+      .def(
+          "get_context",
+          [](InferenceTrace& self, const std::string mode) -> py::object {
+            auto context = (mode == "opentelemetry") ? self.Context() : "";
+            if (context != "") {
+              return py::str(context);
+            }
+            return py::none();
+          },
+          py::arg("mode").none(false) = "triton");
```

Comment:
I'm in favor of removing mode for now: https://github.com/triton-inference-server/server/pull/6985/files#r1525487116

Reply:
updated